__tests__/app.test.js

```diff
@@ -196,5 +196,45 @@ describe("Ticket 5.", () => {
     .expect(400)
   })
 })
+describe("Ticket 7, add comments", () => {
+  test("Adds a new comment onto the specified article.", () => {
+    return request(app)
+    .post("/api/articles/2/comments")
+    .send({"username" : "butter_bridge", "body" : "test comment"})
+    .expect(200)
+    .then(({body}) => {
+      expect(body.comment[0].author).toBe("butter_bridge")
+    })
+  })
+  test("Returns a comment object with the correct properties." ,() => {
+    return request(app)
+    .post("/api/articles/2/comments")
+    .send({"username" : "butter_bridge", "body" : "test comment"})
+    .expect(200)
+    .then(({body}) => {
+      expect(body.comment.length > 0).toBe(true)
+      body.comment.forEach((comment) => {
+        expect(comment).toHaveProperty("comment_id")
+        expect(comment).toHaveProperty("body")
+        expect(comment).toHaveProperty("votes")
+        expect(comment).toHaveProperty("author")
+        expect(comment).toHaveProperty("article_id")
+        expect(comment).toHaveProperty("created_at")
+      })
```

Comment:
For the above two tests, the response we get back from this endpoint should be one element. When we only get the one element back generally we should just send back that object, rather than an array with one object in it. You will need to refactor your tests to reflect this and just check the object that comes back. Ideal to check for the type of values too rather than just the keys.

Reply:
Thanks Rich!